.github/workflows/promote-release.yml

```diff
@@ -13,11 +13,8 @@ on:
         required: true
         default: false
       channel:
-        description: If this is a prerelease, is it alpha or beta?
-        type: choice
-        options:
-          - alpha
-          - beta
+        type: string
+        description: Release channel for prereleases
```

Comment:
Could you please rebase and get rid of these workflow changes?

Reply:
This appears to be a bug in GitHub. All of these .github/workflow files are identical on both branches.